App.tsx

```diff
@@ -1,5 +1,19 @@
 import AppNavigator from '@/navigation/AppNavigator';
 
+import React, { useState, useEffect } from 'react';
+import { Text, View, StyleSheet } from 'react-native';
+import { useFonts, Poppins_400Regular, Poppins_700Bold } from '@expo-google-fonts/poppins';
+// Change the fonts from above into the fonts you downloaded, change the default font name to poppins
+
 export default function App() {
-  return <AppNavigator />;
+  let [fontsLoaded] = useFonts({
+    Poppins_400Regular, //Input the fonts you want to use here
+    Poppins_700Bold 
+  });
+
+  if (!fontsLoaded) {
+    return null;
```

Comment:
I'm not sure how react native would react to the root component being null, so maybe just make this an empty <View>